web/templates/gear_menu_popover.hbs

```diff
@@ -0,0 +1,174 @@
+<ul class="navbar-dropdown-menu" id="gear-menu-dropdown" aria-labelledby="settings-dropdown" data-simplebar>
+    <div class="org-info">
+        <li class="org-name">{{realm_name}}</li>
+        <li class="org-url">{{realm_url}}</li>
+        {{#if is_self_hosted }}
+            <li class="org-version">
+                <a href="#about-zulip" class="navigate-link-on-enter">{{version_display_string}}</a>
+            </li>
+            {{#if server_needs_upgrade }}
+            <li class="org-upgrade">
+                <a href="https://zulip.readthedocs.io/en/stable/production/upgrade.html" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t 'Upgrade to the latest release' }}</a>
+            </li>
+            {{/if}}
+        {{else}}
+            <li class="org-plan hidden-for-spectators">
+                {{#if is_plan_limited }}
+                    <a href="/plans/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">Zulip Cloud Free</a>
+                {{else if is_plan_standard}}
+                    <a href="/plans/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">Zulip Cloud Standard</a>
+                {{else if is_plan_standard_sponsored_for_free}}
+                    <a href="/plans/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">Zulip Cloud Standard (sponsored)</a>
+                {{/if}}
+            </li>
+        {{/if}}
+        {{#if (and is_plan_limited is_owner) }}
+        <li class="org-upgrade">
+            <a href="/upgrade/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t "Upgrade to {standard_plan_name}" }}</a>
+        </li>
+        {{/if}}
+        {{#if is_plan_limited }}
+            {{#if is_education_org }}
+            <li class="org-upgrade">
+                <a href="/upgrade/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t 'Request education pricing' }}</a>
+            </li>
+            {{else if (not is_business_org) }}
+            <li class="org-upgrade">
+                <a href="/upgrade/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t 'Request sponsorship' }}</a>
+            </li>
+            {{/if}}
+        {{/if}}
+    </div>
+    <div class="navbar-dropdown-divider" role="presentation"></div>
```

Comment:
Inner `</ul>` closes, then outer `</li>`, then a new `<li class="something">` to take the place of the divider div, with an inner `<ul>` again wrapping this set of `<li>` elements.

Reply:
Those outer `<li>` elements with a meaningful, possibly sared class name could then take `border-top` to create the line that currently is it's own `<div>`

Reply:
Done. used border-bottom since that works better with personal menu.